docs/style/entities/work.md

```diff
@@ -0,0 +1,85 @@
+content. A work is distinct from another if it is revised, adapted, translated
+or reconstructed. Content is not considered a distinct work if it only
+contains minor variations such as spelling or typographical differences.
+
+Sometimes content is integral to a work and is not considered distinct from
+it. In general, chapters of a novel are *not* considered to be their own
+distinct works. Poems, essays, articles and short stories *are* generally
+considered distinct works.
+
+## Naming
+
+The name of a work should be its canonical title, in the language it was
+written by its author, or translator of translated works. Additional names,
+such as descriptive titles, known variations or transliterations should be
+included as aliases.
+
+The language of a work’s name can differ from the language of the actual work.
+For example, a work written in English can have a Latin name. The name of the
+work should be capitalized based on the rules of the language of its name.
+
+## Untitled works
+
+If a work does not have a title, the name of the work should be the first line
+or sentence of the work’s text, enclosed in square brackets. Do not apply
+capitalization rules; retain the case of the original text. Examples:
+
+- [At that hour when all things have repose]
+- [Rash mortal! stay thy feet and look around]
+
+The sort name of an untitled work should be the same name, omitting the square
+brackets.
+
+## Writer
+
+The primary relationship of a work is to its main author, aka its *writer*;
+every work should have at least one writer.
+
+A work’s writer can be a special-purpose author in specific cases:
+
+- *Anonymous writer*. If a work cannot be attributed to a specific writer
+(e.g. published anonymously or writer attribution has been irrecoverably lost)
+it should be related to
+[[anonymous]](https://bookbrainz.org/author/fd47e471-a994-4ed9-bf52-531d5f184dd3).
+- *Unknown writer*. If the author of a work is currently unknown, but can
+potentially be determined at a later date, it should be related to
+[[unknown]](https://bookbrainz.org/author/6c1b8f55-4c7e-4739-bfa2-1979da4c68e1). 
+- *Oral tradition*. If a work has been preserved in the oral tradition (e.g.
+received, preserved, and transmitted orally from one generation to another) it
+should be related to
+[[oral tradition]](https://beta.bookbrainz.org/author/415a1f7c-a793-4107-9f2f-c38caf15116d).
+
+## Translations
+
+Each translation of a work is distinct from the original work as well as from
+any other translations. A translated work should be related to the original
+work as a translation.
+
+A translated work should have a *writer* relationship to the original work’s
+author, and a *translator* relationship to its translator. If the translator
+is unknown or unidentified, the translator should be
+[[unknown]](https://bookbrainz.org/author/6c1b8f55-4c7e-4739-bfa2-1979da4c68e1).
+
+Any notable information about a translation should be documented in its annotation.
+
```

Comment:
We could also add an example of original work with its  translated work(s).
You get first dibs for choosing a work you like :p

Reply:
Done.